### PR TITLE
Feature view and activity utils

### PR DESCRIPTION
**Pull request by shanmugasanthosh7**
Added travis.yml and more utils functions

### CODE REVIEW

File: droidils/src/main/java/com/aptus/droidils/utils/ContextUtils.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import android.app.Activity
+import android.content.Context
+import android.support.v7.app.AppCompatActivity
+import android.view.inputmethod.InputMethodManager
+
+/**
+ * Hide Keyboard
+ */
+fun Context.hideKeyboard() {
+    val inputManager = this.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
```
Comment (shanmugasanthosh7) [inline droidils/src/main/java/com/aptus/droidils/utils/ContextUtils.kt]: InputMethodManager change into a new function.

**Review by shanmugasanthosh7 — COMMENTED** [review]
Check ContextUtils.

File: droidils/src/main/java/com/aptus/droidils/utils/ContextUtils.kt (hunk 1 above)
Comment (luffalabs) [inline droidils/src/main/java/com/aptus/droidils/utils/ContextUtils.kt:14]: Change InputMethodManager as new function.

**Review by luffalabs — APPROVED** [review]
Add documentation for newly added functions.

**Review by luffalabs — APPROVED** [review]
this looks good.

**Pull request merged**